src/index.ts

```diff
@@ -90,6 +96,16 @@ export interface GetInstanceConfigOptions {
 export type GetInstanceConfigResponse = [IInstanceConfig];
 export type GetInstanceConfigCallback = NormalCallback<IInstanceConfig>;
 
+export type GetInstanceConfigOperationsOptions = PagedOptionsWithFilter;
+export type GetInstanceConfigOperationsResponse = PagedResponse<
+  IOperation,
+  instanceAdmin.spanner.admin.instance.v1.IListInstanceConfigOperationsResponse
+>;
+export type GetInstanceConfigOperationsCallback = RequestCallback<
```

Comment:
I think this should be a `PagedCallback`, as `GetInstanceConfigOperations` returns a paged result

Reply:
Changed it to PagedCallback to have a more specific callback here (`RequestCallback<T, R>` where R does not extend from void also transforms to `PagedCallback<T, R>`)